packages/trip-form/src/util.js

```diff
@@ -107,52 +206,60 @@ function getTransitCombinedModeOptions(icons, modes, selectedModes) {
   );
 }
 
-function getExclusiveModeOptions(icons, modes, selectedModes) {
+/**
+ * Returns the exclusive mode options.
+ * @param {*} ModeIcon The icon component for rendering.
+ * @param {*} modes The available modes to choose from.
+ * @param {*} selectedModes An array of string that lists the modes selected for a trip query.
+ */
+function getExclusiveModeOptions(ModeIcon, modes, selectedModes) {
   const { exclusiveModes } = modes;
 
   return supportedExclusiveModes
-    .filter(mode => exclusiveModes && exclusiveModes.includes(mode.mode))
-    .map(modeObj => ({
-      id: modeObj.mode,
-      selected:
-        !selectedModes.some(isTransit) && selectedModes.some(modeObj.isActive),
+    .filter(({ mode }) => exclusiveModes && exclusiveModes.includes(mode))
+    .map(({ isActive, label, mode }) => ({
+      id: mode,
+      selected: !selectedModes.some(isTransit) && selectedModes.some(isActive),
       showTitle: false,
       text: (
         <span>
-          <ModeIcon icons={icons} mode={modeObj.mode} /> {modeObj.label}
```

Comment:
Is this whitespace preserved?
`<ModeIcon icons={icons} mode={modeObj.mode} /> {modeObj.label}`

Reply:
Good catch. No it is not.